### PR TITLE
Add beta/stable channel support

### PR DESCRIPTION
**Pull request by tobiasKaminsky**
How to test
- run locally via composer start
- curl --request GET --url 'http://localhost:1234/?oem=Nextcloud&version=3.4.0&platform=linux'
- test
  - without channel
  - with channel
  - upgrade from stable to beta version
  - send beta version, shall not offer downgrade, but only next stable

Signed-off-by: tobiasKaminsky <tobias@kaminsky.me>

### CODE REVIEW

File: config/config.php [hunk 1]
```diff
@@ -21,31 +21,40 @@
...
-	$mac_url = $url . 'Mac/Installer/';
+	$stable_linux_url = $url . 'Linux/';
+	$stable_windows_url = $url . 'Windows/';
+	$stable_mac_url = $url . 'Mac/Installer/';
 } else {
-	$url = 'https://github.com/nextcloud/desktop/releases/download/v' . $ver . '/';
-	$linux_url = $url;
-	$windows_url = $url;
-	$mac_url = $url;
+	$stableUrl = 'https://github.com/nextcloud/desktop/releases/download/v' . $stableVersion . '/';
```
Comment (mgallien) [inline config/config.php:39]: I was sure we were using the nextcloud-releases org for that
so can you use `https://github.com/nextcloud-releases/desktop/releases/download/` as the base URL ?

**Review by mgallien — APPROVED** [review]
thanks !

**Comment by tobiasKaminsky** [comment]
Yeah, I cannot wait to see it in action :)

**Review by mgallien — APPROVED** [review]

**Pull request merged**